components/prize_pool/wikis/arenaofvalor/prize_pool_custom.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- @Liquipedia
+-- wiki=arenaofvalor
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {8, 4, 2}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	local prizePool = PrizePool(args):create()
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	return prizePool:build()
+end
+
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
+		placement.placeStart
+	)
+
+	lpdbData.publishertier = Variables.varDefault('tournament_publishertier', '')
+	lpdbData.extradata.publisherpremier = Variables.varDefault('tournament_publisher_major') and 'true' or ''
```

Comment:
Does it need to be adjuted to **lpdbData.publishertier = Variables.varDefault('tournament_publisher_major', '')** instead?

I'm assuming **riotpremier** variable is due to a paste leftover in the row, given the old row-based hasnt been touched since 2019 pretty likely a lot of stuff that shouldn't be there, but is actually there. (Since the game isnt a Riot title, so no page uses **riotpremier=true** anyway)

Reply:
If it's not used it shouldn't be added to the custom